src/controller/AutoCommissioner.cpp

```diff
@@ -378,6 +378,10 @@ CHIP_ERROR AutoCommissioner::CommissioningStepFinished(CHIP_ERROR err, Commissio
                                  "Verify DAC certificate chain and certification declaration to ensure spec rules followed.");
                 }
             }
+            else if (report.Is<CommissionErrorInfo>())
```

Comment:
This won't happen because it's nested in the if statement checking that the stage completed is the attestation verification stage. You'll only get this error response back if you've just completed the kArmFailsafe stage.